lib/babe/build.go

```diff
@@ -166,19 +167,20 @@ func (b *Session) buildBlockExtrinsics(slot Slot) ([]*transaction.ValidTransacti
 			return nil, err
 		}
 
-		// if ret == 0x0001, there is a dispatch error; if ret == 0x01, there is an apply error
-		if ret[0] == 1 || bytes.Equal(ret[:2], []byte{0, 1}) {
-			// TODO: specific error code checking
-			log.Error("[babe] build block apply extrinsic", "error", ret, "extrinsic", extrinsic)
+		// if ret != 0 there was an error
+		if ret[0] != byte(0) {
```

Comment:
does this still work for the  ret == 0x0001 case?

Reply:
the 0x0001 case is handled at line 307.

Reply:
but it won't enter this statement and `determineError` unless the first byte != 0, but for the 0x0001 case, the first byte is 0, so it wouldn't enter here

Reply:
Ok, right, I've updated this.